src/main/java/com/non/model/User.java

```diff
@@ -14,49 +13,145 @@ public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.UUID)
     private UUID id;
-    private Timestamp create_time;
-    private Timestamp update_time;
-    private Integer created_by;
-    private Integer updated_by;
+    private Timestamp createTime;
+    private Timestamp updateTime;
+    private Integer createdBy;
+    private Integer updatedBy;
 
     @Column(unique = true, nullable = false)
     private String username;
-    private String first_name;
-    private String last_name;
+    private String firstName;
+    private String lastName;
 
     @Column(unique = true, nullable = false)
     private String email;
 
     @Column(columnDefinition = "boolean default false")
-    private boolean is_deleted;
+    private boolean isDeleted;
 
-    @Column(columnDefinition = "boolean default true")
-    private boolean active;
+    private boolean isActive = true;
 
     @Enumerated(EnumType.STRING)
     private ROLE role;
 
-    public enum ROLE {
-        ADMIN,
-        MENTOR,
-        MENTEE
-    }
-
     public User() {
     }
 
-    public User(String username, String first_name, String last_name, String email, ROLE role) {
+    public User(String username, String firstName, String lastName, String email, ROLE role) {
         this.username = username;
-        this.first_name = first_name;
-        this.last_name = last_name;
+        this.firstName = firstName;
+        this.lastName = lastName;
         this.email = email;
         this.role = role;
     }
 
-    public User(String username, String first_name, String last_name, String email) {
+    public User(String username, String firstName, String lastName, String email) {
         this.username = username;
-        this.first_name = first_name;
-        this.last_name = last_name;
+        this.firstName = firstName;
+        this.lastName = lastName;
         this.email = email;
+        this.role = ROLE.MENTEE;
+    }
+
+    public UUID getId() {
```

Comment:
@sanskar-19  remove getter setter and use @Data also see @getter @setter etc

Reply:
done